modules/hic-tre-cert.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 # Install HIC CA certificate
-sudo tee /usr/local/share/ca-certificates/hic-tre.crt <<EOF
+sudo mkdir -p /usr/local/share/ca-certificates/hic
```

Comment:
FYI you can use [install](https://man7.org/linux/man-pages/man1/install.1.html) if you want to save a line of typing :smile: 
`install -d /path/to/dir -m MODE`
`install -d /path/to/dir -m MODE -g GROUP -o OWNER`

Reply:
oooooh i forgot that was a thing, very nice